charts/incubator/zammad/questions.yaml

```diff
@@ -0,0 +1,172 @@
+# Include{groups}
+portals: {}
+questions:
+# Include{global}
+# Include{workload}
+# Include{workloadDeployment}
+
+# Include{replicas1}
+# Include{podSpec}
+# Include{containerMain}
+
+# Include{containerBasic}
+# Include{containerAdvanced}
+
+  - variable: zammad
+    group: App Configuration
+    label: Zammad
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: zammad
+          label: Zammad Configuration
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+              - variable: zammad_autowizardjson
+                label: Autowizard JSON
+                description: Allows you to provide initial configuration data for your instance. See the docs for more information.
+                schema:
+                  type: string
+                  default: ""
+        - variable: webconcurrency
+          label: Web Concurrency Configuration
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+              - variable: webcon_instances
+                label: Web Concurrency (Instances)
+                description: Allows spawning n workers to allow more simultaneous connections for Zammad's web interface. If you fill this out then fill out the rest of the Web Concurrency Configuration section.
+                schema:
+                  type: string
+                  default: ""
+              - variable: webcon_sessionjobs
+                label: Concurrent Session Jobs
+                description: Allows specifying the amount of instances of the session worker to run at a time. Increasing this can speed up background jobs (like the scheduler) when too many users are on Zammad at once. Generally speaking, it should only be useful to adjust this setting if you have more than 40 active users at a time.
+                schema:
+                  type: int
+                  default: 0
+              - variable: webcon_schedprocesses
+                label: Independent Process for Scheduled Jobs
+                description: Allows you to enable or disable independent processes for jobs such as LDAP syncs. This can free up the Zammad background worker for other tasks when some scheduled tasks are fairly long.
+                schema:
+                  type: boolean
+                  default: false
+              - variable: webcon_delayedjobs
+                label: Processes for Delayed Jobs
+                description: Allows you to specify the amount of processes that should work on delayed jobs. Increasing this can improve issues with delayed jobs stacking up in your system. You may want to try to use `Concurrent Session Jobs` before though. This option can be very CPU intensive. The maximum value is 16.
+                schema:
+                  type: int
+                  default: 0
```

Comment:
Are any of those values are required? eg I can leave it blank?

Reply:
Good call, I addressed this in latest commit. The integers and booleans should be required, and the string for web concurrency (instances) isn't required.